scripts/run-pelorus-e2e-tests

```diff
@@ -0,0 +1,177 @@
+    local cmd="$*"
+    # Let's print what is happening in the subshell
+    set -x
+    timeout "$timeout" bash -c "until ${cmd}; do sleep ${sleep_time}; done"    
+    set +x
+}
+
+function print_help() {
+    printf "\nUsage: %s [OPTION]... -v [DIR]\n\n" % "$0"
+    printf "\tStartup:\n"
+    printf "\t  -h\tprint this help\n"
+    printf "\n\tOptions:\n"
+    printf "\t  -b\tbranch of the mig-demo-apps\n"
+    printf "\t  -v\tpath to virtualenv DIR\n"
+
+    exit 0
+}
+
+# We use exported functions instead of aliases, so they are available
+# in subshell. This is required for timeout.
+set -a
+# shellcheck disable=SC2269
+PELORUS_NAMESPACE="${PELORUS_NAMESPACE}"
+function ogn() { oc get --namespace "${PELORUS_NAMESPACE}" "$@"; }
+function ogns() { oc get --namespace "${PELORUS_NAMESPACE}" svc "$@"; }
+function ornds() { oc rollout status --namespace ${PELORUS_NAMESPACE} deployments "$@"; }
+function owpr() { oc wait pod --for=condition=Ready -n ${PELORUS_NAMESPACE} -l pelorus.konveyor.io/exporter-type="$*"; }
+set +a
+
+### Options
+OPTIND=1
+while getopts "h?b:v:" option; do
+    case "$option" in
+    h|\?) print_help;;
+    b)    demo_branch=$OPTARG;;
+    v)    venv_dir=$OPTARG;;
+    esac
+done
+
+if [ -z "${venv_dir}" ]; then
+    VENV="${DEFAULT_VENV}"
+else
+    VENV="${venv_dir}"
+fi
+
+if [ -z "${demo_branch}" ]; then
+    demo_branch="master"
+fi
+
+
+
+### MAIN
+# Create download directory
+DWN_DIR=$(TMPDIR="${VENV}" mktemp -d -t "${TMP_DIR_PREFIX}XXXXX") || exit 2
+
+echo "Temporary directory created: ${DWN_DIR}"
+
+# Cleanup download directory on exit
+trap 'cleanup_and_exit' INT TERM EXIT
+
+download_file_from_url "https://raw.githubusercontent.com/konveyor/mig-demo-apps/$demo_branch/apps/todolist-mongo-go/pelorus/values.yaml" "ci_values.yaml" "${DWN_DIR}"
+download_file_from_url "https://raw.githubusercontent.com/konveyor/mig-demo-apps/$demo_branch/apps/todolist-mongo-go/mongo-persistent.yaml" "mongo-persistent.yaml" "${DWN_DIR}"
+
+# Create namespace where pelorus and grafana, prometheus operators will get deployed 
+oc create namespace "${PELORUS_NAMESPACE}"
+
+# Modify downloaded files
+sed -i.bak 's/your_org/konveyor/g' "${DWN_DIR}/mongo-persistent.yaml"
+
+# Show what has been modified:
+diff -uNr "${DWN_DIR}/mongo-persistent.yaml" "${DWN_DIR}/mongo-persistent.yaml.bak"
+
+# From now on, exit if something goes wrong
+set -e
+
+# Ensure we are in the top-level directory of pelorus project
+pushd "${SCRIPT_DIR}/../"
+
+# Apply config maps for the exporters
+oc apply -f "charts/pelorus/configmaps"
+
+helm install operators charts/operators --namespace pelorus --debug --wait --wait-for-jobs
+
+# Wait for grafana and prometheus deployments to be rolled out
+retry 5m 1s ornds prometheus-operator
+retry 5m 1s ornds grafana-operator-controller-manager
+
+helm install pelorus charts/pelorus --namespace pelorus --debug --wait --wait-for-jobs
+
+# check final deployment
+retry 5m 5s ogns grafana-service
```

Comment:
I'm happy w/ this as is.. but to be VERY transparent..
Add.. 
printf("oc get namespace grafana-service)
retry 5m 5s ogns grafana-service
printf("oc get namespace prometheus-operated)
retry 5m 5s ogns prometheus-operated
etc.

and you don't have to worry about the alias :)

Reply:
Added as part of each function, so now every caller will have it